pkg/config/config.go

```diff
@@ -60,6 +61,20 @@ func InitConfigs(clusterConfig *clusterconfig.Config, operatorMetadata *clusterc
 	OperatorMetadata = operatorMetadata
 }
 
+func getClusterConfigFromConfigMap() (clusterconfig.Config, error) {
+	configMapData, err := K8s.GetConfigMapData("cluster-config")
+	if err != nil {
+		return clusterconfig.Config{}, err
+	}
+	clusterConfig := clusterconfig.Config{}
+	errs := cr.ParseYAMLBytes(&clusterConfig, clusterconfig.FullManagedValidation, []byte(configMapData["cluster.yaml"]))
+	if errors.FirstError(errs...) != nil {
+		return clusterconfig.Config{}, errors.FirstError(errs...)
+	}
```

Comment:
I would prefer to have the `cr.ParseYAMLBytes` function to return a single error. You can move the `errors.FirstError` inside